test/unit/test_encoder.py

```diff
@@ -17,10 +17,10 @@
 from sagemaker_containers import content_types, encoders
 
 
-@patch('numpy.load', lambda x: 'loaded %s' % x)
+@patch('numpy.load', lambda x: np.array([42]))
 @patch('sagemaker_containers.encoders.BytesIO', lambda x: 'byte io %s' % x)
 def test_npy_to_numpy():
```

Comment:
Could you test deserialization of other dtypes? The dtypes of the serialized numpy array and deserialized numpy array should match.